examples/specs/window_running_average.vl.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A line graph showing how the average rating for how the MPG has changed over the years. The red line shows the average, and is layered on top of all the raw data.",
+  "data": {"url": "data/cars.json"},
+  "transform": [
+    {"filter": "datum.Miles_per_Gallon !== null"},
+    {"timeUnit": "year", "field": "Year", "as": "year"},
+    {
+      "window": [{
+        "op": "mean",
+        "field": "Miles_per_Gallon",
+        "as": "Average_MPG"
+      }],
+      "frame": [null, null],
+      "groupby": ["year"]
```

Comment:
I think you have to sort by year without groupBy and make frame [-2, 2] to make it 5 year moving average. 

The current setup is simply annual average.

Reply:
You're right I need to update this example because it is no longer a running average. However I cannot figure out why, but there are multiple values for each year if I use something in the future (i.e. `[-2,2]`), however if I use all the values before this year `[null, 0]` there are no problems.

So I updated the example to show a running average of the MPG of all the cars over the years. Which I believe is still relevant because you can see how out of all the cars on the road for a given year, the average tends to get higher.

Reply:
> However I cannot figure out why, but there are multiple values for each year if I use something in the future (i.e. [-2,2]), however if I use all the values before this year [null, 0] there are no problems.

I think we better figure out otherwise it might mean we don't support a very common use case.  (Feel free to ask in Vega channel for more info.)

Reply:
> So I updated the example to show a running average of the MPG of all the cars over the years. Which I believe is still relevant because you can see how out of all the cars on the road for a given year, the average tends to get higher.

This makes sense, but it's a different example.  So rename?

Reply:
"Accumulative average"?